cli/crates/cli/src/main.rs

```diff
@@ -111,12 +126,24 @@ fn try_main(args: Args) -> Result<(), CliError> {
                 process::exit(exitcode::OK);
             });
 
+            // TODO: what's the story for dev and the toml config?
```

Comment:
No config for now. Also, we're moving the start command to a new place in my PR:

```
grafbase federated start
```

We're going to do a bigger split here in the near future:

https://linear.app/grafbase/issue/GB-6147/separate-federatedstandalone-into-subcommands

Reply:
ok, I'll leave a FIXME in place for future reference and leave a note